src/main.js

```diff
@@ -170,13 +172,23 @@ function saveCover() {
 }
 
 function displaySavedCovers() {
-  var savedCoversSectionHTML = ``;
+  savedCoversSection.innerHTML = ``;
   for (var i = 0; i < savedCovers.length; i++) {
-    savedCoversSectionHTML += `<section class="mini-cover">
-    <img class="mini-cover" src=${savedCovers[i].cover} alt="No Image Available">
-    <h2 class="cover-title">${savedCovers[i].title}</h2>
-    <h3 class="tagline">A tale of <span class="tagline-1">${savedCovers[i].tagline1}</span> and <span class="tagline-2">${savedCovers[i].tagline2}</span></h3>
+    savedCoversSection.innerHTML += 
+    `<section class="mini-cover">
+      <img class="cover-image" id=${savedCovers[i].id} src="${savedCovers[i].cover}" alt="No Image Available">
+      <h2 class="cover-title">${savedCovers[i].title}</h2>
+      <h3 class="tagline">A tale of <span class="tagline-1">${savedCovers[i].tagline1}</span> and <span class="tagline-2">${savedCovers[i].tagline2}</span></h3>
     </section>`
   }
-  savedCoversSection.innerHTML = savedCoversSectionHTML;
 }
+
+function removeCover(event) {
+  var something = event.target.id;
+  for (var i = 0; i < savedCovers.length; i++) {
+    if (`${savedCovers[i].id}` === something) {
+      savedCovers.splice(i, 1);
+    }
+  }
+  displaySavedCovers();
+}
```

Comment:
I think git hub is encouraging us to add an extra line at the bottom that is just empty

Reply:
Thank you for catching that! It's been fixed and should be all good to go! :)